app/controllers/prx_auth/rails/sessions_controller.rb

```diff
@@ -44,7 +54,7 @@ def create
         redirect_to after_sign_in_path_for(current_user)
       else
         clear_nonce!
-        redirect_to auth_error_sessions_path(error: 'verification_failed')
+        redirect_to auth_error_sessions_path(error: params[:error] || 'unknown_error')
```

Comment:
Spent awhile debugging an issue, and realized `verification_failed` wasn't the error coming from ID itself, in the params hash.

So I added it to the JS snippet, and passed it through to the UI here.

Reply:
Nice!